go/vt/mysqlctl/mysqld.go

```diff
@@ -647,7 +650,7 @@ func (mysqld *Mysqld) Init(ctx context.Context, cnf *Mycnf, initDBSQLFile string
 	// Start mysqld. We do not use Start, as we have to wait using
 	// the root user.
 	if err = mysqld.startNoWait(ctx, cnf); err != nil {
-		log.Errorf("failed starting mysqld (check mysql error log %v for more info): %v", cnf.ErrorLogPath, err)
+		log.Errorf("failed starting mysqld: %v %s", err, readTailOfMysqldErrorLog(cnf.ErrorLogPath))
```

Comment:
Let's add another newline to set the log intro apart from the error message:

```suggestion
		log.Errorf("failed starting mysqld: %v\n%s", err, readTailOfMysqldErrorLog(cnf.ErrorLogPath))
```